docs/config/templates/app/indexPage.template.html

```diff
@@ -142,17 +142,19 @@ <h4 class="search-results-group-heading">{{ key }}</h4>
           </div>
         </div>
       </nav>
-      <nav id="navbar-notice" class="navbar navbar-fixed-top">
-        <div class="navbar-inner">
-          <div class="container">
-            <p class="site-notice">
-              AngularJS is now in Long Term Support (LTS) mode: <a href="https://docs.angularjs.org/misc/version-support-status">Find out more</a>. Explore End Of Life (EOL) options <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
-            </p>
-          </div>
-        </div>
-      </nav>
       <nav id="navbar-sub" class="sup-header navbar navbar-fixed-top" scroll-y-offset-element ng-cloak>
         <div class="container main-grid main-header-grid">
+        <p class="site-notice">
+              AngularJS support has officially ended as of January 2022.
+              <a href="https://docs.angularjs.org/misc/version-support-status">
+                See what ending support means
+              </a> and
+              <a href="https://goo.gle/angularjs-end-of-life">
+                read the end of life announcement
+              </a>.<br>
+              Visit <a href="https://angular.io">https://angular.io</a> for the actively supported
```

Comment:
```suggestion
              Visit <a href="https://angular.io">angular.io</a> for the actively supported
```
The `https://` may be unnecessary in the banner, where space is somewhat limited.

Reply:
Removed `https://` from the link label 👍